### PR TITLE
Added sample python app with Helm chart and Dockerfile

### PR DESCRIPTION
**Pull request by zr-msft**

### CODE REVIEW

File: samples/python/getting-started/webfrontend/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM python:3
+
+WORKDIR /python/webfrontend
+
```
Comment (DrEsteban) [inline samples/python/getting-started/webfrontend/Dockerfile]: nit: extra spaces :)

File: samples/python/getting-started/webfrontend/webfrontend.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from flask import Flask, request, send_from_directory
+app = Flask(__name__, static_url_path='')
+
+@app.route('/public/<path:path>')
+def static_files(path):
+    return send_from_directory('public',path)
```
Comment (DrEsteban) [inline samples/python/getting-started/webfrontend/webfrontend.py]: >p [](start = 40, length = 1)

nit: add space between params

File: samples/python/getting-started/webfrontend/webfrontend.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+from flask import Flask, request, send_from_directory
```
Comment (DrEsteban) [inline samples/python/getting-started/webfrontend/webfrontend.py]: Copyright on file?

File: samples/python/getting-started/webfrontend/public/app.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+var app = angular.module('myApp', ['ngRoute']);
+
+app.controller('MainController', function($scope, $http) {
+
+    $scope.messages = [];
+    $scope.sayHelloToServer = function() {
+        $http.get("/api?_=" + Date.now()).then(function(response) {
```
Comment (DrEsteban) [inline samples/python/getting-started/webfrontend/public/app.js]: >Date.now() [](start = 30, length = 10)

Why is this being added as a query param?  It doesn't look like it's being read/included by the backend?

File: samples/python/getting-started/webfrontend/webfrontend/templates/tests/test-connection.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+kind: Pod
+metadata:
+  name: "{{ include "webfrontend.fullname" . }}-test-connection"
+  labels:
+    app.kubernetes.io/name: {{ include "webfrontend.name" . }}
+    helm.sh/chart: {{ include "webfrontend.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+  annotations:
+    "helm.sh/hook": test-success
```
Comment (DrEsteban) [inline samples/python/getting-started/webfrontend/webfrontend/templates/tests/test-connection.yaml]: >"helm.sh/hook" [](start = 4, length = 14)

Just curious: Was this file generated by the default Helm3 template?

File: samples/python/getting-started/webfrontend/Dockerfile (hunk 1 above)
Comment (zr-msft) [reply samples/python/getting-started/webfrontend/Dockerfile]: good catch, fixed

File: samples/python/getting-started/webfrontend/webfrontend.py (hunk 2 above)
Comment (zr-msft) [reply samples/python/getting-started/webfrontend/webfrontend.py]: fixed

File: samples/python/getting-started/webfrontend/webfrontend/templates/tests/test-connection.yaml (hunk 5 above)
Comment (zr-msft) [reply samples/python/getting-started/webfrontend/webfrontend/templates/tests/test-connection.yaml]: yes, this was generated by Helm 3

**Pull request merged**